### DISCUSSION TITLE
Can i check if basic package of WABA supported using 360dialog?

### DISCUSSION THREAD

**Discussion by harismateen** [chatwoot/chatwoot#8185] [Feature Requests]
### Is your feature or enhancement related to a problem? Please describe.

Hi, can i check if this WhatsApp API supported as on below API docs?

API DOCS - https://docs.360dialog.com/partner/integration/waba-integration

Host: [waba-v2.360dialog.com](http://waba-v2.360dialog.com/)

### Describe the solution you'd like

Hi, can i check if this WhatsApp API supported as on below API docs?

API DOCS - https://docs.360dialog.com/partner/integration/waba-integration

Host: [waba-v2.360dialog.com](http://waba-v2.360dialog.com/)

**Comment by pranavrajs** [comment]
Moving this to feature requests. We don't support the latest APIs in 360Dialog right now.

**Comment by harismateen** [comment]
@pranavrajs Can I check if any updates on this to use the 360 Dialog Cloud API?
FYI - 360dialog are removing the On-premise APIs later this year (as their partner sent us the email and asking us to migrate to 360Dialog Cloud API instead of On-premise APIs)
